message TestMessage
{
    optional int32 a = 1;
    optional int32 b = 2;
}

